### PR TITLE
Coverity 1216302,1022033,1022032,1022031: State While Revalidate

### PR DESCRIPTION
**Pull request by vmamidi**
Stale While Revalidate Coverity bug fixes
CID 1216302 (#1 of 1): Unchecked return value (CHECKED_RETURN)
CID 1022033 (#1 of 1): Unchecked return value (CHECKED_RETURN)
CID 1022032 (#1 of 1): Unchecked return value (CHECKED_RETURN)
CID 1022031 (#1 of 1): Unchecked return value (CHECKED_RETURN)

### CODE REVIEW

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/526/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/536/

**Comment by atsci** [comment]
AU check *failed*! https://ci.trafficserver.apache.org/job/autest-github/523/

**Comment by atsci** [comment]
Intel CC build *failed*! https://ci.trafficserver.apache.org/job/icc-github/652/

**Comment by atsci** [comment]
FreeBSD11 build *failed*! https://ci.trafficserver.apache.org/job/freebsd-github/2219/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/780/

**Comment by atsci** [comment]
Linux build *failed*! https://ci.trafficserver.apache.org/job/linux-github/2115/

**Comment by vmamidi** [comment]
[approve ci]

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/541/

**Comment by atsci** [comment]
AU check *failed*! https://ci.trafficserver.apache.org/job/autest-github/528/

**Comment by atsci** [comment]
FreeBSD11 build *failed*! https://ci.trafficserver.apache.org/job/freebsd-github/2224/

**Comment by atsci** [comment]
Intel CC build *failed*! https://ci.trafficserver.apache.org/job/icc-github/657/

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/531/

**Comment by atsci** [comment]
Linux build *failed*! https://ci.trafficserver.apache.org/job/linux-github/2120/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/785/

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/532/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/542/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/529/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/658/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/786/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/2121/

**Comment by atsci** [comment]
FreeBSD11 build *successful*! https://ci.trafficserver.apache.org/job/freebsd-github/2225/

File: plugins/experimental/stale_while_revalidate/stale_while_revalidate.c [hunk 1]
```diff
@@ -660,12 +666,15 @@ main_plugin(TSCont cont, TSEvent event, void *edata)
...
-    TSHandleMLocRelease(buf, loc, warn_loc);
-    TSHandleMLocRelease(buf, TS_NULL_MLOC, loc);
+    if (TSHttpTxnClientRespGet(txn, &buf, &loc) == TS_SUCCESS) {
+      TSMimeHdrFieldCreateNamed(buf, loc, TS_MIME_FIELD_WARNING, TS_MIME_LEN_WARNING, &warn_loc);
+      TSMimeHdrFieldValueStringInsert(buf, loc, warn_loc, -1, HTTP_VALUE_STALE_WARNING, strlen(HTTP_VALUE_STALE_WARNING));
+      TSMimeHdrFieldAppend(buf, loc, warn_loc);
+      TSHandleMLocRelease(buf, loc, warn_loc);
+      TSHandleMLocRelease(buf, TS_NULL_MLOC, loc);
+    } else {
+      TSError("[stale_while_revalidate][%s] Error while getting response from txn", __FUNCTION__);
```
Comment (SolidWallOfCode) [inline plugins/experimental/stale_while_revalidate/stale_while_revalidate.c]: `TSError("[%s] ....", PLUGIN_NAME)`.

**Comment by atsci** [comment]
clang format *failed*! https://ci.trafficserver.apache.org/job/clang-format-github/538/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/551/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/536/

**Comment by atsci** [comment]
Linux build *failed*! https://ci.trafficserver.apache.org/job/linux-github/2128/

**Comment by atsci** [comment]
Intel CC build *failed*! https://ci.trafficserver.apache.org/job/icc-github/665/

**Comment by atsci** [comment]
FreeBSD11 build *failed*! https://ci.trafficserver.apache.org/job/freebsd-github/2232/

**Comment by atsci** [comment]
clang-analyzer build *failed*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/794/

**Pull request merged**